client/src/components/__tests__/authWrapper.test.js

```diff
@@ -5,6 +5,23 @@ import { createTestingPinia } from '@pinia/testing';
 import { useAuthStore } from '../../stores/auth';
 import { nextTick } from 'vue';
 
+vi.mock('vue-router/auto-routes');
+vi.mock('vue-router', async () => {
+	const actual = await vi.importActual('vue-router');
+	return {
+		...actual,
+		useRoute: vi.fn(() => {
+			return routeMock;
+		}),
+	};
+});
+
+const routeMock = {
+	meta: {
+		auth: true,
+	},
+};
+
```

Comment:
From the perspective of someone not knowledgeable about Vue, I might like some more comments here, just because while I can at what this code is doing (mocking a route, I suppose?) I'd feel more comfortable if comments were there providing a description.

Reply:
Entirely legitimate! As I state in my message below, my general preference is for the explicit over the concise, which comes with its fair share of drawbacks. And my recommendations here are informed by my lack of understanding of Vue, which leaves me desirous for more clarity in comments, even if such clarity would be considered redundant by a seasoned Vue developer. 

I also see what you're talking about with regards to follow-up documentation. While I'd like to say that I put all my comments up as I'm developing the functions, I have plenty of evidence to know this is not the case 😅.